scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl

```diff
@@ -71,9 +67,15 @@ workflow GvsCreateVATfromVDS {
                 output_basename = vcf_filename
         }
 
+        call RemoveDuplicatesFromSitesOnlyVCF {
+            input:
+                sites_only_vcf = SelectVariants.output_vcf,
+                ref = reference
+        }
```

Comment:
So this is scattered now but it looks like it's still using the same resources (memory, disk, cpu etc) that it was when it was unscattered? Perhaps the monitoring script could confirm whether all those resources are still required.